webdriver-spec.html

```diff
@@ -742,6 +742,9 @@
 
   <p>Otherwise, let <var>parameters</var> be null.</p>
 
+ <li><p>If a <a>navigation</a> is currently underway <a>wait for
+  navigation to complete</a>.
```

Comment:
I think there is language for it, and you can find an example under [traversing the history by a delta](https://html.spec.whatwg.org/#traverse-the-history-by-a-delta), where this paragraph seems relevant:

> If there is an ongoing attempt to navigate specified browsing context that has not yet _matured_ (i.e. it has not passed the point of making its Document the active document), then cancel that attempt to navigate the browsing context.

And [‘matured’](https://html.spec.whatwg.org/#concept-navigate-mature) seems to be a defined concept too.

So if we want to have this for _every_ command (in the WG we only talked having it for navigation and interaction commands) we ought to change this language to something like

> If there is an ongoing attempt to _navigate_ the _current browsing context_ that has not yet _matured_, _wait for navigation to complete_.